src/main/scala/Week1/ChatBot.scala

```diff
@@ -0,0 +1,30 @@
+package Week1
+
+object ChatBot extends App {
+  val name = "eric"
+  val cannedResponse =  new Array[String](2)
+  cannedResponse(1) = "I don’t know and I’m too lazy to look it up..."
+  cannedResponse(2) = "I don’t know, let me google it up for you"
+  var questionCounter = 1
+
+  while (true) {
+    var input = scala.io.StdIn.readLine()
+
+    if (checkIfQuestion(input) != "") println(checkIfQuestion(input))
```

Comment:
checkIfQuestion(input) != "" could be replaced with !checkIfQuestion(input).isEmpty